### PR TITLE
Know Your Air functionality

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)
- Restores the KYA feature.
- Changes the iOS logo
- Disables using clear traffic while fetching data from our endpoints.
- Upgrades the kotlin version.
- Builds release appbundle (v2.0.1(..5)) and ipa (v2.0.2) for android and iOS.
- Removes unused asset files.
- Fixes multiple bug.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Follow instructions in the [README.md](https://github.com/airqo-platform/AirQo-frontend/tree/MOB-55-v-2-0-1-user-feedback/mobile#airqo-mobile-app) to setup your environment. Interact with the KYA feature.

#### What are the relevant tickets?
- [MOB-43](https://airqoteam.atlassian.net/browse/MOB-43)

#### Screenshots (optional)

### CODE REVIEW

File: mobile/android/app/src/main/res/xml/network_security_config.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <network-security-config>
-    <domain-config cleartextTrafficPermitted="true">
+    <domain-config cleartextTrafficPermitted="false">
```
Comment (NoahNsimbe) [inline mobile/android/app/src/main/res/xml/network_security_config.xml:3]: Disabled use of non-secure TCP communications while fetching data from APIs

File: mobile/android/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
```
Comment (NoahNsimbe) [inline mobile/android/build.gradle:2]: Upgraded kotlin version to enable building appbundles with flutter 2.10.2.

File: mobile/lib/models/kya.dart [hunk 3]
```diff
@@ -42,7 +63,14 @@ class Kya {
       debugPrint('$exception\n$stackTrace');
     }
 
-    return Kya('', '', '', [], 0.0);
+    return Kya(
```
Comment (NoahNsimbe) [inline mobile/lib/models/kya.dart:66]: substituting using positional parameters with named parameters

File: mobile/lib/models/measurement.dart [hunk 4]
```diff
@@ -151,10 +157,4 @@ extension ParseMeasurement on Measurement {
     }
     return false;
   }
-
-  String getTempValue() {
-    var tempValue = temperature.value.toStringAsFixed(2);
-
-    return '$tempValue\u2103';
-  }
 }
```
Comment (NoahNsimbe) [inline mobile/lib/models/measurement.dart:160]: code formatting. Reorganising methods

File: mobile/lib/on_boarding/setup_complete_screeen.dart [hunk 5]
```diff
@@ -55,8 +55,8 @@ class SetUpCompleteScreenState extends State<SetUpCompleteScreen> {
   }
 
   Future<void> initialize() async {
-    Future.delayed(const Duration(seconds: 4), () async {
-      await Navigator.pushAndRemoveUntil(context,
+    Future.delayed(const Duration(seconds: 4), () {
+      Navigator.pushAndRemoveUntil(context,
```
Comment (NoahNsimbe) [inline mobile/lib/on_boarding/setup_complete_screeen.dart:59]: removing unnecessary asynchronous calls

File: mobile/lib/screens/for_you_page.dart [hunk 6]
```diff
@@ -58,11 +53,11 @@ class _ForYouPageState extends State<ForYouPage>
                     onTap: (value) {
                       if (value == 0) {
                         setState(() {
-                          isWeekly = true;
+                          analytics = true;
```
Comment (NoahNsimbe) [inline mobile/lib/screens/for_you_page.dart:56]: Use appropriate variable names

File: mobile/lib/screens/settings_page.dart [hunk 7]
```diff
@@ -81,20 +81,6 @@ class _SettingsPageState extends State<SettingsPage> {
...
-      //     await showSnackBar(context, 'failed to delete account. '
-      //         'Try again later');
-      //     Navigator.pop(dialogContext);
-      //   }
-      // Navigator.pop(dialogContext);
-      // await _appService.logOut(context);
-      // await Navigator.pushAndRemoveUntil(context,
-      //     MaterialPageRoute(builder: (context) {
-      //       return const SignupScreen(false);
-      //     }), (r) => false);
```
Comment (NoahNsimbe) [inline mobile/lib/screens/settings_page.dart:97]: Deleting useless comments

File: mobile/lib/services/app_service.dart [hunk 8]
```diff
@@ -194,6 +196,15 @@ class AppService {
...
 
+  Future<void> fetchKya() async {
+    try {
+      var kyas = await _cloudStore.getKya(_customAuth.getUserId());
+      await _dbHelper.insertKyas(kyas);
+    } catch (exception, stackTrace) {
+      debugPrint('$exception\n$stackTrace');
+    }
+  }
+
```
Comment (NoahNsimbe) [inline mobile/lib/services/app_service.dart:207]: Persisting KYA locally

File: mobile/lib/services/app_service.dart [hunk 9]
```diff
@@ -396,6 +408,18 @@ class AppService {
...
+      await isConnected();
+      await Future.wait([
+        _dbHelper.updateKya(kya),
+        _cloudStore.updateKyaProgress(_customAuth.getUserId(), kya)
+      ]);
+    } else {
+      await _dbHelper.updateKya(kya);
+    }
+  }
+
```
Comment (NoahNsimbe) [inline mobile/lib/services/app_service.dart:422]: Updating KYA user progress

**Review by 123MwanjeMike — APPROVED** [review]

**Review by 123MwanjeMike — APPROVED** [review]

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**